### ISSUE TITLE
Plug-in does not work at all

### ISSUE THREAD

**Issue by Owen61** [Camelcade/Perl5-IDEA#1752]
Intellij 2017.2.4 running on Windows 7 Enterprise. The perl scripts run on a Linux server (mounted via Samba).

To install the plug-in, I followed steps in the Wiki. My working directory is labelled with _Perl5 libraries_ in the explorer pane. But when I open the perl script, it is a mess:
- It thinks the hash-bang at the start of the file is a mistake (#!/usr/bin/perl)
- Comments are interpreted as code (underlined nouns, same font as code)
- There is no colour-coding anywhere
- When typing, there is no autocompletion
- No automatic indenting

So basically, nothing works. It is worse than Notepad.

Is the problem that I am editing on Windows a script that will run on Linux?

**Comment by hurricup** [comment]
Isn't this #1721 ?

**Comment by Owen61** [comment]
Thank you - that was the solution. To re-cap:

- File/settings
- Editor/File types
- In _Recognized File Types_ find _Perl Scripts_ (add it if necessary)
- In _Registered Patterns_, add `*.pl` (perhaps _Prolog is squatting on the extension - if so, take it back)
- Apply/OK

Sorry, I just assumed `*.pl` would be detected automatically.

P.S. Might be an idea to add this to the front of the Wiki?

**Comment by hurricup** [comment]
Planning to fix this with #1724
